website/blog/2022-11-11-pact-plugin-framework-launch.md

```diff
@@ -13,8 +13,8 @@ hide_table_of_contents: false
 
 Today we share the news that your favourite contract testing framework comes Pact with more power! The new Pact Plugin Framework completes the missing piece to help development teams apply contract testing all their API technologies and use cases.  
 
-:::note
-[Register](https://docs.pact.io/events/plugins-framework-launch) for an upcoming webinar to learn more.
+:::Watch
```

Comment:
can you change this back to `:::note` please

https://deploy-preview-201--docs-pact-io.netlify.app/blog/2022/11/11/pact-plugin-framework-launch#pact-with-more-power--introducing-the-pact-plugin-framework

the note tag gets picked up and turned into a nice section :)

https://docusaurus.io/docs/next/markdown-features/admonitions

Reply:
done :)